redaxo/src/core/lib/console/sync_assets.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ * 
+ * @auth staabm
+ *
+ * @internal
+ */
+class rex_command_sync_assets extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sync files of /assets with addons/assets core/assets folders');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $io = $this->getStyle($input, $output);
+
+        foreach(rex_package::getInstalledPackages() as $package) {
+            if ($package instanceof rex_addon) {
+                $feAssetsPath = rex_path::addonAssets($package->getName());
+                $beAssetsPath = rex_path::addon($package->getName(), 'assets/');

```

Comment:
Man könnte hier auch `$package->getAssetsPath()` und `$package->getPath('assets')` nutzen. Dann braucht man keine Unterscheidung nach Addons/Plugins (siehe dazu auch Kommentar im `else`).

Reply:
Zu den Variablennamen: `fe` und `be` kann hier eventuell verwirren, da im Rex-Sprech Frontend und Backend eher was anderes meinen, und die Assets gerade vorwiegend fürs Backend, nicht fürs Frontend verwendet werden.
Oder genauer: Für das Frontend des Backends ;)

Reply:
der pfad unter assets ist z.b. `assets/addons/be_style/plugins/redaxo`, d.h. der assets path vom addon enthält die assets aller plugins

Reply:
Aber im eigentlichen Addon-Pfad liegen die Plugin-Assets doch woanders, und zwar in:
`src/addons/foo/plugins/bar/assets` 
und nicht in:
`src/addons/foo/assets/plugins/bar`